### PR TITLE
#171177795 Return fully info about who liked or disliked

### PR DESCRIPTION
**Pull request by william000000**
#### What does this PR do?
 - Displays more detailed result which includes who liked or dislike
#### Description of Task to be completed?
-  Join booking table and like table
- Display more about who liked or dislike
#### How should this be manually tested?
- `git clone https://github.com/andela/team-odd-bn-backend.git`
- `cd team-odd-bn-backend`
- git checkout ch-user-like-dislike-171177795
- `npm install`
- Make sure your `.env` file is updated.
- Drop existing migration `npm run dropTables`
- Update migration `npm run db-migrate`
- Populate the seed data `npm run db-seed-dev`
- `npm run dev-start` check API in development 
> Create booking `localhost:3000/api/v1/accommodations/{tripId}/likes`.

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#171177795](https://www.pivotaltracker.com/story/show/171177795)
#### Screenshots (if appropriate)
image

#### Questions:
N/A

### CODE REVIEW

**Review by hezronkimutai — CHANGES_REQUESTED** [review]
Please fill in the PR description @william000000

File: src/controllers/BookingController.js [hunk 1]
```diff
@@ -23,6 +23,8 @@ class BookingController {
       await NotificationService.newBookingNotification(req);
       return Response.successMessage(req, res, 'You have booked an accommodation facility successfully', result, 201);
     } catch (error) {
+      console.log(error);
+
```
Comment (NiyoEric) [inline src/controllers/BookingController.js]: Remove console.

**Review by NiyoEric — CHANGES_REQUESTED** [review]
I 've tested likes/dislikes functionality and it is working as expected. Please take look the feedback I left

**Review by hezronkimutai — APPROVED** [review]

**Review by hezronkimutai — COMMENTED** [review]
Fix the failling tests @william000000

**Review by NiyoEric — APPROVED** [review]

**Review by hezronkimutai — APPROVED** [review]

**Review by victkarangwa — APPROVED** [review]
I think you are good to go!

**Pull request merged**